DragaliaAPI/Features/Event/EventDropService.cs

```diff
@@ -165,6 +165,19 @@ QuestData quest
         return dropList;
     }
 
+    private bool IsEligibleForT3Drops(QuestData quest, EventData evt)
+{
+    // Check for specific event names
+    if (evt.EventName == "Accursed Archives" || evt.EventName == "Dream Big Under the Big Top")
+    {
+        return true;
+    }
+
+    // Default behavior for other events
+    return quest.VariationType >= VariationTypes.Hard;
```

Comment:
```suggestion
    return quest.VariationType > VariationTypes.Hard;
```
`(variation - VariationTypes.Hard) / 2d` meant that everything above VariationTypes.Hard would drop T3 items since `(VariationTypes.Hard - VariationTypes.Hard) / 2d  = 0 / 2d = 0`